doc/classes/EditorSettings.xml

```diff
@@ -924,6 +924,9 @@
 		<member name="run/output/font_size" type="int" setter="" getter="">
 			The size of the font in the [b]Output[/b] panel at the bottom of the editor. This setting does not impact the font size of the script editor (see [member interface/editor/code_font_size]).
 		</member>
+		<member name="run/output/max_lines" type="int" setter="" getter="">
+			Maximum amount of lines to show at any one time in the Output panel.
```

Comment:
```suggestion
			Maximum number of lines to show at any one time in the Output panel.
```
Number for countable things, amount for uncountable, generally

Reply:
Fixed.